zipkin-storage/cassandra3/src/main/java/zipkin/storage/cassandra3/CassandraUtil.java

```diff
@@ -96,14 +97,17 @@ public static int durationIndexBucket(long ts_micro) {
       return Collections.emptyList();
     }
     checkArgument(request.serviceName != null, "serviceName needed with annotation query");
-    Set<String> annotationKeys = new LinkedHashSet<>();
```

Comment:
set dedupes. maybe I should have put in a comment, but I think we need this (probably also a unit test to show it works)